Wangscape/noise/module/CornerCombinerBase.h

```diff
@@ -4,6 +4,8 @@ namespace noise
 {
 namespace module
 {
+// Default power for the noise::module::CornerCombinerBase module.
```

Comment:
Yep. I don't know what documentation tool Wangscape will use (probably doxygen), but when doc generation is added to the build process, comments like this will have to be formatted for compatibility.